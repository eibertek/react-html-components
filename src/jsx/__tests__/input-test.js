import React from 'react';
import Input from '../input.jsx';

describe('Link', () => {
    it('changes the class when hovered', () => {
        expect(true).toBe(true);
    });
});
