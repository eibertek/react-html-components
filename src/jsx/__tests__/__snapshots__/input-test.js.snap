exports[`Input with listeners matches snapshot 1`] = `
<div
  className="input-field col s12">
  <input
    className="validate "
    id="test"
    onChange={[Function bound _onChange]}
    onMouseEnter={[Function bound _onMouseEnter]}
    onMouseLeave={[Function bound _onMouseLeave]}
    placeholder=""
    type="text" />
  <label
    dangerouslySetInnerHTML={
      Object {
        "__html": "test label"
      }
    }
    htmlFor="test" />
</div>
`;

exports[`Input with value matches snapshot 1`] = `
<div
  className="input-field col s12">
  <input
    className="validate "
    id="test"
    onChange={[Function bound _onChange]}
    onMouseEnter={[Function bound _onMouseEnter]}
    onMouseLeave={[Function bound _onMouseLeave]}
    placeholder=""
    type="text" />
  <label
    dangerouslySetInnerHTML={
      Object {
        "__html": "test label"
      }
    }
    htmlFor="test" />
</div>
`;

exports[`Input without listeners matches snapshot 1`] = `
<div
  className="input-field col s12">
  <input
    className="validate "
    id="test"
    onChange={[Function bound _onChange]}
    onMouseEnter={[Function bound _onMouseEnter]}
    onMouseLeave={[Function bound _onMouseLeave]}
    placeholder=""
    type="text" />
  <label
    dangerouslySetInnerHTML={
      Object {
        "__html": "test label"
      }
    }
    htmlFor="test" />
</div>
`;
